spidermonkey/jsconfig.mk

```diff
@@ -82,7 +82,7 @@ ifeq ($(OS_ARCH), WINNT)
   NSPR_SHARED    = nspr20/$(NSPR_VERSION)/$(NSPR_OBJDIR)
 endif
 NSPR_VERSIONFILE = $(NSPR_LOCAL)/Version
-NSPR_CURVERSION := $(shell cat $(NSPR_VERSIONFILE))
+#NSPR_CURVERSION := $(shell cat $(NSPR_VERSIONFILE))
```

Comment:
Any particular reason?

Reply:
Yes; as noted in the proposed changes, the NSPR `Version` file never gets created. This line attempts to `cat` a non-existent file twice which causes 2 unnecessary errors during the build process of SpiderMonkey. This line is not required and there is no harm in commenting it.